### PR TITLE
qa_crowbarsetup: Support SES4 for SOC7

### PR DESCRIPTION
**Pull request by toabctl**
SOC7 should support SES4 so setup the correct repos. (UNTESTED)

### CODE REVIEW

**Comment by vuntz** [comment]
cc @tserong

**Comment by toabctl** [comment]
tested and it's **NOT** working. So needs more work.

**Comment by tserong** [comment]
AFAICT the SUSE-Enterprise-Storage-4-{Pool,Updates} repos aren't on the clouddata server (I'm not sure yet if that's the only problem).

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -357,6 +349,11 @@ function addsles12sp2testupdates
     # [[ $hacloud = 1 ]] && add_mount "SLE12-SP2-HA-Updates-test" \
     #     $distsuseip':/dist/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP2:/x86_64/update/' \
     #     "$tftpboot_repos12sp2_dir/SLE12-SP2-HA-Updates-test/"
+    if isrepoworking SUSE-Enterprise-Storage-4-Updates-test ; then
+        [ -n "$deployceph" -a iscloudver 7plus ] && add_mount "SUSE-Enterprise-Storage-4-Updates-test" \
+            $distsuseip":/dist/ibs/SUSE:/Maintenance:/Test:/Storage:/4:/$arch/update/" \
```
Comment (tserong) [inline scripts/qa_crowbarsetup.sh]: The log reports

````
+(qa_crowbarsetup.sh:353) addsles12sp2testupdates(): '[' -n '' -a iscloudver 7plus ']'
./scripts/qa_crowbarsetup.sh: line 353: [: too many arguments
````

Might want to be something like `iscloudver 7plus && [ -n "$deployceph" ] && add_mount ...` (untested)

**Comment by tserong** [comment]
That gating run doesn't seem to have `deployceph` set. Does that matter?

The failure seems to be because a node deployed didn't reach the ready state. Is it possible to get a log from that node?

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: hm. that's just copied from a couple of lines below.

**Comment by toabctl** [comment]
> AFAICT the SUSE-Enterprise-Storage-4-{Pool,Updates} repos aren't on the clouddata server (I'm not sure yet if that's the only problem).

Fixed

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (tserong) [inline scripts/qa_crowbarsetup.sh]: the one a couple of lines below is probably broken too then ;-)

**Comment by toabctl** [comment]
@tserong gate passed now. any other comments on this?

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: fixed I think :)

**Comment by tserong** [comment]
LGTM

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -337,14 +334,18 @@ function addsles12sp1testupdates
...
     fi
     if isrepoworking SUSE-Enterprise-Storage-3-Updates-test ; then
-        [ -n "$deployceph" -a iscloudver 7plus ] && add_mount "SUSE-Enterprise-Storage-3-Updates-test" \
-            $distsuseip":/dist/ibs/SUSE:/Maintenance:/Test:/Storage:/3:/$arch/update/" \
-            "$tftpboot_repos12sp1_dir/SUSE-Enterprise-Storage-3-Updates-test/"
+        if [ -n "$deployceph" ] && iscloudver 7plus; then
+            add_mount "SUSE-Enterprise-Storage-3-Updates-test" \
+                      $distsuseip":/dist/ibs/SUSE:/Maintenance:/Test:/Storage:/3:/$arch/update/" \
+                      "$tftpboot_repos12sp1_dir/SUSE-Enterprise-Storage-3-Updates-test/"
+        fi
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: Shouldn't we just drop this bit?

**Pull request merged**